packet/loadbalancers.go

```diff
@@ -63,6 +67,18 @@ func (l *loadBalancers) init(k8sclient kubernetes.Interface) error {
 // we do this via metallb, not directly, so none of this works... for now.
 
 func (l *loadBalancers) GetLoadBalancer(ctx context.Context, clusterName string, service *v1.Service) (status *v1.LoadBalancerStatus, exists bool, err error) {
+	// If a service has had a loadbalancer address applied via the alternative reconcilliation loop then we can return it as the
```

Comment:
This can actually be removed now (the status is updated in the other reconciliation)

Reply:
I don't understand the comment. If we can remove this, why are we adding it?

Reply:
More importantly, I have 2 questions:

1. are we sure that these do not conflict? Is the "right" way to set the `LoadBalancerIP` here via the `Status` and not there via the `Spec.LoadBalancerIP`?
1. Are we sure that setting the status does not mess up metallb-system? We need to ensure that keeps working correctly. It is worth digging into how the speaker there works, to make sure that it is not expecting to find the IP and set the status, and perhaps ignore it if the status is set already?

Reply:
The speaker does exactly the same thing. Once it has done its work it will also update the status. The code above was left in from my work last night. Didn’t realise I’d left it in when I commited this morning.

Reply:
1. yes.. the status is reflect the current "state" of the load balancer.. the spec reflects the "expected state" of the load balancer

2. metallb will do the same thing, once  it does it's thing it will do an `updateStatus(sevicev1)`

Reply:
1. So we need both? What is the "right place" to set the `Spec` and what is the "right place" to set the `Status`?
1. for metallb, see my comments below.

Reply:
1. We set both in the original place we set the `Spec` as mentioned first.. this code was put in the wrong place by me in testing.